### PR TITLE
7903658 Document build/make.sh

### PR DESCRIPTION
**Pull request by LudwikJaniuk**
Add a note in building.md about build/make.sh, explaning that you can use it to rebuild faster.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [CODETOOLS-7903658](https://bugs.openjdk.org/browse/CODETOOLS-7903658): Document build/make.sh (**Enhancement** - P4)

### Reviewers
 * [Christian Stein](https://openjdk.org/census#cstein) (@sormuras - Committer) ⚠️ Review applies to [85ee2590](https://git.openjdk.org/jtreg/pull/182/files/85ee259077e9b048e58cfc7bd01e900913dd531d)
 * [Jonathan Gibbons](https://openjdk.org/census#jjg) (@jonathan-gibbons - **Reviewer**) ⚠️ Review applies to [85ee2590](https://git.openjdk.org/jtreg/pull/182/files/85ee259077e9b048e58cfc7bd01e900913dd531d)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtreg.git pull/182/head:pull/182` \
`$ git checkout pull/182`

Update a local copy of the PR: \
`$ git checkout pull/182` \
`$ git pull https://git.openjdk.org/jtreg.git pull/182/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 182`

View PR using the GUI difftool: \
`$ git pr show -t 182`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtreg/pull/182.diff">https://git.openjdk.org/jtreg/pull/182.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jtreg/pull/182#issuecomment-1938286602)

### CODE REVIEW

**Review by sormuras — APPROVED** [review]
Looks reasonable to me.

File: doc/building.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ variables used to define the location of dependencies to be downloaded.
 These are documented in _make/build.sh_ and are normally specified in
 _make/build-support/version-numbers_ and _make/build-support/*/version-numbers_ .
 
+### Rebuilding faster with `build/make.sh`
+
+After you have run `build.sh` once, if you wish to rebuild after making some
```
Comment (jonathan-gibbons) [inline doc/building.md:45]: Suggest giving the full path for `make/build.sh`

File: doc/building.md [hunk 2]
```diff
@@ -40,6 +40,11 @@ variables used to define the location of dependencies to be downloaded.
 These are documented in _make/build.sh_ and are normally specified in
 _make/build-support/version-numbers_ and _make/build-support/*/version-numbers_ .
 
+### Rebuilding faster with `build/make.sh`
+
+After you have run `build.sh` once, if you wish to rebuild after making some 
+changes, you can run `build/make.sh` which should be significantly faster.
+
```
Comment (jonathan-gibbons) [inline doc/building.md:47]: Suggest: 

```
you can just run `build/make.sh`, which skips the steps to download and build the dependencies, 
and so should be significantly faster.
```

**Review by jonathan-gibbons — APPROVED** [review]
Thanks for this; this is good for now.

It would be tweak to tweak the script the remember the MAKE_ARGS but still allow them to be overridden. And then we should document that.

File: doc/building.md (hunk 2 above)
Comment (LudwikJaniuk) [reply doc/building.md:48]: Done

**Pull request closed without merging**